debian/rules

```diff
@@ -1,61 +1,18 @@
 #! /usr/bin/make -f
 #
 # 	Nicotine+
+#	Copyright (C) 2021 Nicotine+ Team
 #	Copyright (C) 2016 Kip Warner. Released under the GPLv3 or later.
 #
 
 # Output every command that modifies files on the build system...
 DH_VERBOSE = 1
 export DH_OPTIONS=-v
 
-# Set flags for how the package will be built. Note separated with white
```

Comment:
I am fine in principal with removing the `get-orig-source` target, but I'd like it if you could verify with a DPM expert (not me) whether this is appropriate here. The reason is that the deprecation notice for the target I think contemplates official Debian packages where downstream packaging and upstream source are in separate locations. So you may want to look into what the best practise is for unofficial packages.

Reply:
Do you know any acquaintance or other person who could verify this?

I'd personally prefer to diverge as little as possible from a future downstream Debianization, minimize version parsing and hardcoding, and keep things simple in general.

Reply:
> Do you know any acquaintance or other person who could verify this?
> 
> I'd personally prefer to diverge as little as possible from a future downstream Debianization, minimize version parsing and hardcoding, and keep things simple in general.

Best guy to ask is the one and only DPM guru I ask whenever I have questions, @pabs3.

Reply:
Also ask in #packaging on irc.oftc.net. That's where I usually find him and others.

Reply:
Thanks! I'll keep this in mind for later.